src/Tribe/Attendee_Registration/Main.php

```diff
@@ -29,11 +29,18 @@ class Tribe__Tickets__Attendee_Registration__Main {
 	 * @return string
 	 */
 	public function get_slug() {
-		$slug = Tribe__Settings_Manager::get_option( 'ticket-attendee-page-slug', false );
-
-		$page = get_page_by_path( $slug );
+		$id = Tribe__Settings_Manager::get_option( 'ticket-attendee-page-id', false );
+
+		// check for old saved slug - since TBD
+		if ( ! empty( $id ) ) {
+			$page = get_page( $id );
+			$slug = $page->post_name;
+		} else {
+			$slug = Tribe__Settings_Manager::get_option( 'ticket-attendee-page-slug', false );
+			$page = get_page_by_path( $slug );
+		}
 
-		if ( empty( $slug ) || ! has_shortcode( $page->post_content, 'tribe_attendee_registration' ) ) {
+		if ( empty( $slug ) || empty( $page ) || ! has_shortcode( $page->post_content, 'tribe_attendee_registration' ) ) {
```

Comment:
Will there be a case where the isn’t both or none: slug/page?

Reply:
eh, trying to avoid calling `$page->post_content` if we don't have a `$page` - let me rework that.